AffiliateWP/affiliatewp-external-referral-links
# issue/33 - Use cookie name getter to get affwp_erl_id cookie name

Issue #33


**Housekeeping**


**Proposed changes**


## Before changes - external page on a Pantheon's edge network
![image](https://user-images.githubusercontent.com/498472/122424851-f18c4080-cf86-11eb-85ab-2f506ca88403.png)

## After changes - external page on a Pantheon's edge network
![image](https://user-images.githubusercontent.com/498472/122425016-18e30d80-cf87-11eb-8268-63226b908a75.png)

## How to test it
1. Use the AffiliateWP branch `issue/3937`
2. Install and activate External Referral Links add-on
3. On `affiliatewp-external-referral-links.php:202` force AffiliateWP version check to be 2.7.1+: `$affwp_version = '2.7.1';` so that it doesn't depend on AffiliateWP version
3. On wp-config add the following server environment variable: `$_ENV['PANTHEON_ENVIRONMENT'] = 1;` to simulate being on Pantheon's edge network
5. Create a fake external landing page (can be on the same site)
6. Open that page not logged in and confirm the external referral link tracking cookie name (`affwp_erl_id`) is prefixed with `wp_`
